parser-typechecker/src/Unison/Codebase/Editor/SlurpResult.hs

```diff
@@ -154,116 +165,158 @@ pretty
+               <$> toList (termExistingConstructorCollisions sr)
+               )
+            ++ (   termLineFor ConstructorExistingTermCollision
+               <$> toList (constructorExistingTermCollisions sr)
+               )
+            ++ (   termLineFor BlockedDependency
+               <$> toList (terms (defsWithBlockedDependencies sr))
+               )
+      in
+        header
+        <> "\n\n"
+        <> P.hiBlack "  Reason"
+        <> "\n"
+        <> P.indentN 2 (P.linesNonEmpty [typeMsgs, termMsgs])
+        <> "\n\n"
+        <> P.indentN
+             2
+             (P.column2 [("Tip:", "Use `help filestatus` to learn more.")])
+    dups = Set.toList (SC.terms (duplicates sr) <> SC.types (duplicates sr))
+    more i =
+      "... "
+        <> P.bold (P.shown i)
+        <> P.hiBlack " more."
+        <> "Try moving these below the `---` \"fold\" in your file."
   in
-  P.sepNonEmpty
-    "\n\n"
-    [ if SC.isEmpty (duplicates sr)
-      then mempty
-      else
-        (if isPast
-            then "⊡ Ignored previously added definitions: "
-            else "⊡ Previously added definitions will be ignored: "
-          )
-          <>  P.indentNAfterNewline 2
-             (  P.wrap
-               $ P.excerptSep' (Just 7) more " " (P.hiBlack . prettyVar <$> dups)
-             )
-    , oks (P.green "I've added these definitions:")
-          (P.green "These new definitions are ok to `add`:")
-          (adds sr)
-    , oks
-      (P.green "I've updated to these definitions:")
-      (P.green
-      $ "These new definitions will replace existing ones of the same name and "
-      <> "are ok to `update`:"
-      )
-      (updates sr)
-    , notOks
-      (P.red "These definitions failed:")
-      (P.wrap $ P.red "These definitions would fail on `add` or `update`:")
-      sr
-    ]
+    P.sepNonEmpty
+      "\n\n"
+      [ if SC.isEmpty (duplicates sr)
+        then mempty
+        else
+          (if isPast
+              then "⊡ Ignored previously added definitions: "
+              else "⊡ Previously added definitions will be ignored: "
+            )
+            <> P.indentNAfterNewline
+                 2
+                 (P.wrap $ P.excerptSep' (Just 7)
+                                         more
+                                         " "
+                                         (P.hiBlack . prettyVar <$> dups)
+                 )
+      , oks (P.green "I've added these definitions:")
+            (P.green "These new definitions are ok to `add`:")
+            (adds sr)
+      , oks
+        (P.green "I've updated to these definitions:")
+        (P.green
+        $ "These new definitions will replace existing ones of the same name and "
+        <> "are ok to `update`:"
+        )
+        (updates sr)
+      , notOks
+        (P.red "These definitions failed:")
+        (P.wrap $ P.red "These definitions would fail on `add` or `update`:")
+        sr
+      ]
```

Comment:
This chunk ended up being massive due to brittnification. 😓 
I think we only changed `okType`, `okTerm`, and `aliases`, but it is hard to know for sure.

Reply:
☝️ Github truncated the chunk displayed here to 100 lines, so you don't really get the full effect.